index.js

```diff
@@ -152,6 +161,41 @@ module.exports.processV2 = (output, origin) => {
   throw 'no url';
 };
 
+module.exports.processV3 = (output, origin) => {
+  const data = JSON.parse(output.toString().trim());
+  const { automatic_captions, formats, fragments, subtitles, url: audio } = data;
+
+  const cookies = data.http_headers && data.http_headers.Cookie || '';
+  const duration = data.duration || 0;
```

Comment:
I think this would be better as `??` personally

Reply:
Same for the rest tbh. They're functionally the same but or'ing two falsy values seems a bit strange